src/main/fc/fc_core.c

```diff
@@ -393,7 +393,7 @@ void processRx(timeUs_t currentTimeUs)
     const throttleStatus_e throttleStatus = calculateThrottleStatus();
 
     if (isAirmodeActive() && ARMING_FLAG(ARMED)) {
-        if (rcCommand[THROTTLE] >= rxConfig()->airModeActivateThreshold) airmodeIsActivated = true; // Prevent Iterm from being reset
+        if (rcData[THROTTLE] >= rxConfig()->airModeActivateThreshold) airmodeIsActivated = true; // Prevent Iterm from being reset
```

Comment:
This seems to be a bug that is affecting the behaviour outside of your proposed change, and should probably be fixed in 3.2. Would you mind opening a separate PR for it?

Well spotted by the way!

Reply:
True. So it's broken altogether for center-stick-3D at the moment?

Reply:
Yes as far as I can tell.  I can't fly with center stick 3d too well...

Reply:
Sounds like a question for @rotcehdnih.